shapepipe/modules/ngmix_package/ngmix.py

```diff
@@ -813,7 +813,6 @@ def do_ngmix_metacal(
 
         weight_map = np.copy(weights[n_e])
         weight_map[np.where(flags[n_e] != 0)] = 0.
-        weight_map[weight_map != 0] = 1
```

Comment:
This will create an issue. Line 859 there is `weight_map *= 1 / sig_noise ** 2` if the weight_map is not equal to 1 it will lead to weird behavior.

Reply:
for theli weights, there are relative weights between pixels, which then need to be rescaled by the sky background variance to become variance maps.  Thomas Erban seemed to think this operation was okay.  are you talking about the weirdness associated with some megaprime weight maps that are not always 1 or 0?